src/main.cpp

```diff
@@ -6083,7 +6083,7 @@ bool static ProcessMessage(const CChainParams& chainparams, CNode* pfrom, string
     }
 
 
-    else if (strCommand == "tx")
+    else if (strCommand == "tx" && !IsInitialBlockDownload(chainparams.GetConsensus()))
```

Comment:
I confirmed that this will not result in any `tx` messages that we request being ignored, because all requests we make that might result in a `tx` message stem from the `pfrom->AskFor(inv)` call that is also skipped.